### PR TITLE
Greeting to Kottans

### PR DESCRIPTION
**Pull request by HannaSyn**
### I've just learned, how to create pull-requests :star_struck: 
Thanks for learning!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1377,3 +1377,10 @@ I am sure it will be an awesome journey!
 # Hello there!
 1st time working via command line, [Vasiliy](https://github.com/Vas-P) =)
 here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-printing.jpg)
+
+---
+# Hello, Kottans! 
+
+I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join community! :smiling_face_with_three_hearts:
+[kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg?1492508022)
```
Comment (Temu4) [inline README.md:1385]: Hi Hanna!
Awesome! 👍 
But if you want to embed an image in markdown you can try to use such template
`image description`

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nazmariam) [reply README.md:1385]: More info [here](https://www.markdownguide.org/basic-syntax/#images)

Comment (Temu4) [reply README.md]: > Hi Hanna!
> Awesome! 👍
> But if you want to embed an image in markdown you can try to use such template
> `image description`

I mean `!` symbol 😄

Comment (HannaSyn) [reply README.md]: Thank you! Got it :sweat_smile:

**Comment by Temu4** [comment]
Well done!

**Pull request merged**